### PR TITLE
Introduce new tenant configuration events

### PR DESCRIPTION
**Pull request by herdt-michael**
Introduce new events that are triggered when changes are made to the tenant configuration.

Signed-off-by: Michael Herdt <Michael.Herdt@bosch.io>

### CODE REVIEW

File: hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/model/JpaTenantConfiguration.java [hunk 1]
```diff
@@ -16,7 +16,19 @@
 import javax.validation.constraints.NotNull;
 import javax.validation.constraints.Size;
 
+import org.eclipse.hawkbit.repository.event.remote.RolloutDeletedEvent;
```
Comment (StefanKlt) [inline hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/model/JpaTenantConfiguration.java:19]: Some of these imports are never used

Comment (herdt-michael) [reply hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/model/JpaTenantConfiguration.java]: Removed.

**Review by StefanKlt — APPROVED** [review]
Looks good, thanks @herdt-michael

**Review by stefbehl — APPROVED** [review]
Looks good. Thanks Michael.

**Pull request merged**